lib/membrane_rtc_engine/tees/tee.ex

```diff
@@ -109,6 +111,24 @@ defmodule Membrane.RTC.Engine.Tee do
     {{:ok, actions}, state}
   end
 
+  @impl true
+  def handle_event(
+        Pad.ref(:input, {_track_id, variant}) = pad,
+        %VoiceActivityChanged{} = vad,
+        _ctx,
+        state
+      ) do
+    state = put_in(state, [:vad, pad], vad)
+
+    # only forward to subscribed routes
+    actions =
+      state.routes
+      |> Enum.filter(fn {_pad, route} -> route.current_variant == variant end)
+      |> Enum.map(fn {pad, _route} -> {:event, {pad, vad}} end)
```

Comment:
That's a nitpick but by a single route I meant a tuple `{destination_pad, config}`. 

Take a look at line 225

```
      Enum.flat_map_reduce(state.routes, state, fn route, state ->
        handle_route(buffer, variant, route, ctx, state)
      end)
```